NewForce-Cohort-9/tabloid-full-stack-full-stack-force
# Fw ticket34 add reaction

Add Reactions
Description
Added Add Reactions function in Post Details to add reactions for a post. [Trello Ticket #35](https://trello.com/c/VU0B7vNY/34-34-add-a-reaction-to-a-post)

Type of Change
-Added new, non-breaking feature.

Testing Instructions
1. git add and git commit whatever you're working on on your branch
2. git checkout main
3. git fetch --a
4. git checkout fw-ticket34-addReaction
5.If you don't see the most recent version of my files, git pull origin fw-ticket34-addReaction
6. Inside of Visual Studio (purple), click the execute with debugger solid green button to launch the app
7. In your terminal, navigate to where your package.json file is in your client folder. Run "npm start" in your terminal.
8. Login as an admin. foo@bar.com for the email parameter. Leave the password text box blank. Click "Login".
9. Click on Posts at the top of the navbar.
10. Once on the posts page, click on any post. In the post details, you should see thumbs up and thumbs down with count next to them. 
11. Click on the thumbs up and check that the thumbs up count goes up. Then, click on thumbs down and check that thumbs up count goes down, while thumbs down count goes up.
12. Then click again on thumbs down to check that the thumbs down count goes down.

13. Therefore you can give a thumbs up or down, and click on a reaction twice to withdraw it.

Checklist:
[x] My code follows the style guidelines of this project
[x] I have performed a self-review of my code
[x] I have commented my code, particularly in hard-to-understand areas
[x] My changes generate no new warnings or errors
[x] I have added test instructions that prove my fix is effective or that my feature works
